packer/linux/scripts/install-buildkite-test-splitter.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+case $(uname -m) in
+x86_64) ARCH=amd64 ;;
+aarch64) ARCH=arm64 ;;
+*) ARCH=unknown ;;
+esac
+
+echo "Creating test-splitter user and group..."
```

Comment:
I'm curious as to whether this is required?

Reply:
Very doubtful. The agent has its own user and group for privilege separation because it is a daemon (more or less) that runs arbitrary jobs. test-splitter seems like a tool that is run as needed by jobs run by the agent, so it would be acceptable to run under the same user and group as the agent.

Reply:
Brill, thanks. Have snipped :+1: